storage/innobase/fil/fil0fil.cc

```diff
@@ -366,7 +366,7 @@ fil_node_t* fil_space_t::add(const char* name, pfs_os_file_t handle,
 /** Open a tablespace file.
 @param node  data file
 @return whether the file was successfully opened */
-static bool fil_node_open_file_low(fil_node_t *node)
+static MY_ATTRIBUTE((warn_unused_result)) bool fil_node_open_file_low(fil_node_t *node)
```

Comment:
All callers of a `static` function must be in the same compilation unit. It’s rather easy to check that the return value is not being ignored by any caller.

If you think that the attribute is useful, I’d put it right before the comment:
```c++
__attribute__((warn_unused_result, nonnull))
/** Open a tablespace file.
@param node  data file
@return whether the file was successfully opened */
static bool fil_node_open_file_low(fil_node_t *node)
```